### DISCUSSION TITLE
Adding SPEDAS to VS Code

### DISCUSSION THREAD

**Discussion by abkeaton** [interactive-data-language/vscode-idl#28] [Q&A]
Hi! I'm new to both IDL and vscode and was told that the idl extension for vscode would be quicker to learn. I have to use the SPEDAS library but I'm not sure how to make it work with vscode. Thanks!

**Comment by znorman-harris** [comment]
Hey there! I'm not quite sure of what the SPEDAS library is, but a couple of things that you'll want to do:

1. Either open up a workspace/folder in VSCode that has the SPEDAS library in it

![image](https://github.com/interactive-data-language/vscode-idl/assets/31664668/1daca247-c882-42fc-a916-a66cf9df5a9a)

2. Or you can change the setting for IDL's search path to include the folder where the SPEDAS library lives.
For IDL, if you have nested folders, you'll want to put a "+" before the path entry to recursively find all of your code. That means path entries look something like `+C:\my\folder\path`.

Here's a screenshot showing the buttons you click to get to IDL's search path:

![image](https://github.com/interactive-data-language/vscode-idl/assets/31664668/0c019c8a-ed30-4d46-b574-80ada64284ad)
